homework 1 and 2/script.js

```diff
@@ -0,0 +1,66 @@
+
+//homework 1. . Create a page which contains a button. By clicking on that button current datetime
+// should appear on the right of the button. Style html by yourself.
+const newDateTime = new Date();
+const navTime = document.querySelector("#navTime");
+
+//buttonShowTime
+const buttonshowTime = document.querySelector("#showTime");
+buttonshowTime.addEventListener("mouseenter", () => {
+  buttonshowTime.style.backgroundColor = "blue"
```

Comment:
because you use the same logic for multiple places you can extract the logic into a function